src/wms/src/main/java/org/geoserver/wms/capabilities/Capabilities_1_3_0_Transformer.java

```diff
@@ -1,4 +1,4 @@
-/* (c) 2014 - 2015 Open Source Geospatial Foundation - all rights reserved
+/* (c) 2014 - 2016 Open Source Geospatial Foundation - all rights reserved
```

Comment:
Yes, https://github.com/geoserver/geoserver/wiki/GSIP-147 has been accepted but the developer guide has not yet been updated. My personal preference is to permit unneeded copyright header updates as it may take a while for the new policy to be adopted in practice. The new policy is there to make life easier for developers; requiring developers to undo unnecessary copyright header updates defeats this.  :-)